### ISSUE TITLE
Font Size Control

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Kiniaru/mobile-rss-reader#32]
```
Think this was raised by another user, so I'm seconding this. Font size 
control for the feedlist because it looks quite big in portrait form as it 
is. Smaller font will mean bigger workspace, and program would look so 
much cooler imho :)

Thanks!

My file version is: midp20_RSSReader-1_11_1.jar

```

Original issue reported on code.google.com by `david...@gmail.com` on 4 Nov 2008 at 1:03

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Thanks for using the software and the enhancement request.  I will try to put 
this 
in the next minor release (1.11.2) if feasible if not the next major release 
(1.12).  I think that J2ME will allow it for the version that you are using 
with 
MIDP 2.0 (midp20_RSSReader-1_11_1.jar).  I would have the font size as an 
option 
that would be used on all lists (headers, river of news, bookmarks) and in the 
future feed items.  

As for the feed that the item came from in river of news, I would do this as an 
option as well from settings as people with small phones may not have enough 
space 
to do this.  One issue with putting the feed name at the beginning of the item 
would 
be if it was long, should I limit it to so many characters.  Let me know your 
thoughts.
```

Original comment by `ibunto...@gmail.com` on 7 Nov 2008 at 7:00
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Hi ibuntonjr,

Thanks for your response. Sounds great and looking forward to that.

About the river of news, having it as option would be great!I do realize that 
it 
gets crowded. Even now without the feed name, the titles are usually too long. 
But 
as mentioned, that's partly due to the big font size now.

Another way to know what feed it is while we scroll down the river of news 
would be 
to have the feedname displayed at the top of the screen somewhere. The title 
would 
change as each update is selected/highlighted. This saves space and would look 
good, 
hopefully not confusing.

Hope you understand what I'm trying to convey, and that it agrees with the way 
you'd 
want it to work. Let me know if I could clarify and offer anymore from here.

Keep up the great work!

Thanks.
David
```

Original comment by `david...@gmail.com` on 7 Nov 2008 at 1:50

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Hi ibuntonjr,

received your update about an unstable version in this thread. can't find that 
attachment or msg anymore. is it available for testing?
```

Original comment by `david...@gmail.com` on 8 Dec 2008 at 2:08

**Comment by GoogleCodeExporter** [comment]
```
Hi,

I was wanting to check it in first, but I will attach the zipped source and 
program 
file.  The change will affect new lists.  To get it for the bookmarks, you need 
to 
exit and come back in again.  Then, it will be for all lists.  I found that the 
emulator had a problem with the fix.  The fix on my cell phone worked OK, so 
some 
devices may or may not have problems.  This also allows for turning on 
wraparound on 
lists and changing sort direction on River of News.

Regards,

Irving
```

Original comment by `ibunto...@gmail.com` on 9 Dec 2008 at 8:36
- Changed state: **Started**

Attachments:
- [midp20_jsr75_dev1_RSSReader-1_11_2.jar](https://storage.googleapis.com/google-code-attachments/mobile-rss-reader/issue-32/comment-5/midp20_jsr75_dev1_RSSReader-1_11_2.jar)
- [midp20_cldc11_jsr75_src.zip](https://storage.googleapis.com/google-code-attachments/mobile-rss-reader/issue-32/comment-5/midp20_cldc11_jsr75_src.zip)

**Comment by GoogleCodeExporter** [comment]
```
Hi Irving,

THanks for this. Seems like you're working intensively on it at the moment, 
keep up 
the good job!

First thing I tried was to import my opml file (saved in my phone). When I 
chose 'Find files' like I used to, it just shows Loading files to import 
from... for 
very long. I just have to press Back and just exit.

Wonder if you encountered this in your test so far. My phone is N95 8GB. I'll 
have 
to revert to the old version now.

All the best!

Thanks.
David
```

Original comment by `david...@gmail.com` on 12 Dec 2008 at 6:21

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Hi David,

Sorry for the bug.  Thanks for being willing to try a development version.  I 
didn't 
test the code for this problem before.  I have fixed it and created a 
Development 2 
release on the main download page.  It also gives an error if a problem is 
encountered, which shouldn't happen anymore.  I also put in the bookmark/feed 
name 
on river of news option as well.  The other part of your request to have the 
title 
changed to the feed name when the item is selected in the list, is harder to do 
as 
J2ME does not you to check this unless you make your own implementation of the 
list.  I'm working on this for 1.12 which is when I will implement similar 
things.

Regards,

Irving

```

Original comment by `ibunto...@gmail.com` on 13 Dec 2008 at 9:02
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed on Beta 1 Version 1.11.2 except setting title based on position in list.

```

Original comment by `ibunto...@gmail.com` on 16 Feb 2009 at 2:37

**Comment by GoogleCodeExporter** [comment]
```
This is fixed on version 1.11.2.
```

Original comment by `ibunto...@gmail.com` on 14 May 2009 at 5:12
- Changed state: **Fixed**
- Added labels: **Milestone-Release1.11.2**, **OpSys-All**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.11.2.
```

Original comment by `ibunto...@gmail.com` on 13 Oct 2009 at 7:43
- Changed state: **Verified**
